src/platform/portduino/PortduinoGlue.cpp

```diff
@@ -100,33 +101,20 @@ void portduinoSetup()
     settingsStrings[displayspidev] = "";
     settingsMap[spiSpeed] = 2000000;
     settingsMap[ascii_logs] = !isatty(1);
+    settingsMap[displayPanel] = no_screen;
+    settingsMap[touchscreenModule] = no_touchscreen;
 
     YAML::Node yamlConfig;
 
     if (configPath != nullptr) {
         std::cout << "Using " << configPath << " as config file" << std::endl;
-        try {
-            yamlConfig = YAML::LoadFile(configPath);
-        } catch (YAML::Exception &e) {
-            std::cout << "Could not open " << configPath << " because of error: " << e.what() << std::endl;
-            exit(EXIT_FAILURE);
-        }
+        loadConfig(configPath);
     } else if (access("config.yaml", R_OK) == 0) {
```

Comment:
TOCTOU bug here and below.    (No obvious security issue.)    A better practice is to just try to open the file and deal with failures from open(2).

Reply:
Good point.